### PR TITLE
Revert translation updates

### PR DESCRIPTION
**Pull request by emteknetnz**
Reverts this part of this commit
https://github.com/silverstripe/silverstripe-realme/commit/0238a241450f0483d728d30a8e801bd840da100e#diff-0df2677d84007a9e9dbf1c06cbc8ee7cc2ceb663efb442203e5206de2efcdf32L6

### CODE REVIEW

**Review by Cheddam — CHANGES_REQUESTED** [review]
The phone number changes are incorrect too - see [RealMe's contact details](https://www.realme.govt.nz/contact-us/).

**Comment by emteknetnz** [comment]
Good spot, have updated

File: lang/en.yml [hunk 1]
```diff
@@ -3,18 +3,22 @@ en:
     ASSERTLOGINBUTTON: 'Share your details with {orgname}'
     AUTHENTICATOR_NAME: 'RealMe Account'
     LOGINBUTTON: Login
+  SilverStripe\RealMe\Authenticator\LoginHandler:
+    LOGINFAILURE: 'Unfortunately we''re not able to log you in through RealMe right now. Please try again shortly.'
```
Comment (Cheddam) [inline lang/en.yml]:
```suggestion
    LOGINFAILURE: "Unfortunately we're not able to log you in through RealMe right now. Please try again shortly."
```

Comment (emteknetnz) [reply lang/en.yml]: Good spot.  There's actually lots of strings in en.yml that had this weird method of single quoting, so I've converted them all to be surrounded with double quotes

File: lang/en.yml [hunk 2]
```diff
@@ -3,36 +3,40 @@ en:
...
+    ERR_CONFIG_ENTITYID_SERVICE_NAME: "The service name '{serviceName}' must be a maximum of 20 characters and not blank for entityID '{entityId}'"
+    ERR_CONFIG_INVALID_AUTHNCONTEXT: "The AuthnContext specified for environment '{env}' is invalid, please check your configuration"
+    ERR_CONFIG_NO_AUTHNCONTEXT: "No AuthnContext specified for environment '{env}'. Specify this in your YML configuration, see the module documentation for more details"
+    ERR_CONFIG_NO_ENTITYID: "No entityID specified for environment '{env}'. Specify this in your YML configuration, see the module documentation for more details"
+    ERR_CONFIG_NO_ORGANISATION_DISPLAY_NAME: "No organisation display name is specified in YML configuration. Ensure the 'metadata_organisation_display_name' value is defined in your YML configuration"
+    ERR_CONFIG_NO_ORGANISATION_NAME: "No organisation name is specified in YML configuration. Ensure the 'metadata_organisation_name' value is defined in your YML configuration"
+    ERR_CONFIG_NO_ORGANISATION_URL: "No organisation URL is specified in YML configuration. Ensure the metadata_organisation_url value is defined in your YML configuration"
```
Comment (Cheddam) [inline lang/en.yml]:
```suggestion
    ERR_CONFIG_NO_ORGANISATION_URL: "No organisation URL is specified in YML configuration. Ensure the 'metadata_organisation_url' value is defined in your YML configuration"
```

File: lang/en.yml [hunk 3]
```diff
@@ -3,36 +3,40 @@ en:
...
-    ERR_CONFIG_NO_AUTHNCONTEXT: 'No AuthnContext specified for environment ''{env}''. Specify this in your YML configuration, see the module documentation for more details'
-    ERR_CONFIG_NO_ENTITYID: 'No entityID specified for environment ''{env}''. Specify this in your YML configuration, see the module documentation for more details'
-    ERR_CONFIG_NO_ORGANISATION_DISPLAY_NAME: 'No organisation display name is specified in YML configuration. Ensure the ''metadata_organisation_display_name'' value is defined in your YML configuration'
-    ERR_CONFIG_NO_ORGANISATION_NAME: 'No organisation name is specified in YML configuration. Ensure the ''metadata_organisation_name'' value is defined in your YML configuration'
-    ERR_CONFIG_NO_ORGANISATION_URL: 'No organisation URL is specified in YML configuration. Ensure the ''metadata_organisation_url'' value is defined in your YML configuration'
+    ERR_CONFIG_ENTITYID: "The Entity ID ('{entityId}') must be https, not be 'localhost, and must contain a valid service name and privacy realm e.g. https://my-realme-integration.govt.nz/p-realm/s-name"
```
Comment (Cheddam) [inline lang/en.yml]:
```suggestion
    ERR_CONFIG_ENTITYID: "The Entity ID ('{entityId}') must be https, not be 'localhost', and must contain a valid service name and privacy realm e.g. https://my-realme-integration.govt.nz/p-realm/s-name"
```

**Review by Cheddam — COMMENTED** [review]
Two more tiny fixes and then we can merge 😅

File: lang/en.yml (hunk 2 above)
Comment (emteknetnz) [reply lang/en.yml]: updated

File: lang/en.yml (hunk 3 above)
Comment (emteknetnz) [reply lang/en.yml]: updated

**Review by Cheddam — APPROVED** [review]

**Pull request merged**